content/applications/inventory_and_mrp/quality/quality_management/quality_checks.rst

```diff
@@ -122,16 +123,34 @@ Quality check on work order
 ---------------------------
 
 To process a quality check for a work order, begin by navigating to :menuselection:`Manufacturing
---> Operations --> Manufacturing Orders`, then select a manufacturing order. Select the
-:guilabel:`Work Orders` tab, then click the :guilabel:`📱 (tablet)` tablet view button for the work
-order that requires the quality check.
+--> Operations --> Manufacturing Orders`. Select an |MO| that includes a work order for which a
+quality check is required.
 
-With tablet view open, complete the steps listed on the left side of the screen until the quality
-check step is reached, then follow the instructions at the top of the screen. If a Pass - Fail check
-is being processed, complete the check by clicking :guilabel:`Pass` or :guilabel:`Fail` at the top
-of the screen. For all other quality check types, a :guilabel:`Next` button appears instead. Click
-it to complete the check and move on to the next step of the work order.
+On the |MO|, select the :guilabel:`Work Orders` tab, and then click the :guilabel:`Open Work Order
+(square with arrow coming out of it)` button on the line of the work order to be processed. On the
+resulting :guilabel:`Work Orders` pop-up window, click the :guilabel:`Open Shop Floor` button to
+open the *Shop Floor* module.
 
-.. image:: quality_checks/work-order-check.png
-   :align: center
-   :alt: A quality check for a work order.
+.. note::
+   For a full guide to the Shop Floor module, see the
+   :doc:`Shop Floor overview <../../manufacturing/shop_floor/shop_floor_overview>` documentation.
+
+When accessed from a specific work order, the *Shop Floor* module opens to the page for the work
+center where the order is configured to be processed, and isolates the work order's card so that no
+other cards are shown.
+
+Process the work order's steps until the quality check step is reached. Click on the step to open a
+pop-up window that details how the check should be completed. After following the instructions,
+click :guilabel:`Validate` to complete the check. Alternatively, if a *Pass - Fail* check is being
+processed, click either the :guilabel:`Pass` or :guilabel:`Fail` button.
+
+It is also possible to complete a quality check by clicking the check box on the right side of the
```

Comment:
Use "checkbox" instead of "check box"
```suggestion
It is also possible to complete a quality check by clicking the checkbox on the right side of the
```